runner.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+#!/usr/bin/python3
+module load nextflow-20.10.0
+module load singularity-3.7.1
+
+INPUT_DIRS=data/
+OUT_DIRS=$(cd $INPUT_DIRS; find * -type d -prune -exec ls -d {} \; |head -1)
+echo $OUT_DIRS
+
+ILLUM_DIRS=data/illumina/
+ILLUM_RES=$(cd $ILLUM_DIRS; ls -td * |head -1)
+
+ARTIC_DIRS=data/artic/
+ARTIC_RES=$(cd $ARTIC_DIRS; find * -type d -prune -exec ls -d {} \; |head -1)
+
+MIDNIGHT_DIRS=data/midnight/
+MIDNIGHT_RES=$(cd $MIDNIGHT_DIRS; find * -type d -prune -exec ls -d {} \; |head -1)
+
+if [ $OUT_DIRS == "illumina" ]; then
+echo "
+     ========================================== 
+     FOUND ILLUMINA SAMPLE DIRECTORY: $ILLUM_RES  
+     ==========================================="
```

Comment:
These echo messages are cool BUT
Is the runner meant to auto run? In that case, the formatting will EAT log space.

Reply:
Yes, when user starts this runner.sh, it will check for new uploads on the /data directory and check whether it is illumina or nanopore (artic or midnight) and also the meta files contain the same files/names (sampleValidation.py). Then it starts to check the metafile has all required information (cli.py). If everything goes well, then this will starts running the nextflow analysis. 
True, it will use the log space, and we can remove this to save the space.

Reply:
removed from the repo.